data/data/agent/systemd/units/agent-extract-tui.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Extract agent-tui at boot
+After=selinux.service
```

Comment:
I think we want to be after [`start-local-registry.service`](https://github.com/openshift/appliance/blob/master/data/services/common/start-local-registry.service) too
(that's a really bad unit name :disappointed:)